src/main/java/fr/sncf/osrd/train/phases/SignalNavigatePhase.java

```diff
@@ -37,16 +37,15 @@
     public transient List<SpeedControllerGenerator> targetSpeedGenerators;
```

Comment:
Lots of style changes here aren't really consistent with the rest of the project. Maybe we could think about a better system here, like exporting an Intellij style config file and pushing it to the git?